src/main/java/com/programmers/springbasic/command/wallet/GetCustomersByVoucher.java

```diff
@@ -0,0 +1,36 @@
+package com.programmers.springbasic.command.wallet;
+
+import static com.programmers.springbasic.constants.MessageConstants.*;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.springframework.stereotype.Component;
+
+import com.programmers.springbasic.command.Command;
+import com.programmers.springbasic.console.ConsoleInputHandler;
+import com.programmers.springbasic.console.ConsoleOutputHandler;
+import com.programmers.springbasic.controller.VoucherController;
+import com.programmers.springbasic.dto.CustomerDto;
+
+@Component
+public class GetCustomersByVoucher implements Command {
```

Comment:
일관성을 위해 Command를 붙여주는 게 좋아보입니다.
```suggestion
public class GetCustomersByVoucherCommand implements Command {
```